src/AutofacSerilogIntegration/ContextualLoggingModule.cs

```diff
@@ -26,11 +27,12 @@ public ContextualLoggingModule()
             _skipRegistration = true;
         }
 
-        internal ContextualLoggingModule(ILogger logger = null, bool autowireProperties = false, bool dispose = false)
+        internal ContextualLoggingModule(ILogger logger = null, bool autowireProperties = false, bool dispose = false, bool onlyKnownConsumers = false)
```

Comment:
Perhaps the name should be `reflectionActivatorsOnly`?

Reply:
I feel this would be a little too specific and breaking encapsulation. But you want to invert it anyway, so this is probably a moot point.